res/controllers/Stanton-DJC-4.midi.xml

```diff
@@ -2578,6 +2578,16 @@
           <normal />
         </options>
       </control>
+      <control>
+        <group>[Library]</group>
+        <key>GoToItem</key>
+        <description>GoToItem - contextual behaviour.</description>
+        <status>0x90</status>
+        <midino>0x47</midino>
+        <options>
+          <normal/>
+        </options>
```

Comment:
does this already cover both decks?

Reply:
well this was more about the fact that from what I can tell, there is one shift button and one vinyl button per deck and this only maps a single message. So this seems to me like you're only mapping one button even though there is one per deck (two per controller).

Reply:
Ideally, GoToItem is mapped to the browse encoder pressed down, but I assume this hardware doesn't have that?

Reply:
We were discussing other option in the AutoDJ thread.
This devices has to rotary/push encoders. @xxxmicrobexxx wanted to map AutoDJ actions to the lower encoder, while I think it would make more sense to have those on the per-deck vinyl buttons and move all library actions to the center section.
tbc